documentation/index.md

```diff
@@ -1,48 +1,33 @@
 ---
 title: FEniCS Documentation
-subtitle: fenicsversion
+subtitle: fenicsxversion
 permalink: /documentation/
 ---
 
-The FEniCSx documentation includes a set of books, collections of documented demo programs and reference manuals.
+The FEniCSx documentation is available at
+[https://docs.fenicsproject.org/](https://docs.fenicsproject.org/).
 
-## The FEniCSx Tutorial
```

Comment:
Would be nice to keep a link to the FEniCSx tutorial on this page somewhere

Reply:
I didn't get past reading about the old tutorial . . . I'll add a link back.